app/src/main/kotlin/com/flxrs/dankchat/data/repo/IgnoresRepository.kt

```diff
@@ -96,6 +96,9 @@ class IgnoresRepository @Inject constructor(
 
         runCatching {
             val blocks = apiManager.getUserBlocks(id) ?: return@withContext
+            if (blocks.data.isEmpty()) {
+                return@withContext
```

Comment:
let's also clear `_twitchBlocks`, just in case

Reply:
Roger roger